src/formats/yarle/utils/templates/default-template.ts

```diff
@@ -1,7 +1,6 @@
 const frontmatterDelimiter = '---\n';
 const sourceBlock = '{source-url-block}source: {source-url}{end-source-url-block}\n';
 const tagBlock = '{tags-yaml-list-block}\ntags: {tags-yaml-list}\n\n{end-tags-yaml-list-block}';
-const titleBlock = '{title-block}# {title}\n\n{end-title-block}';
```

Comment:
Is there an issue corresponding to this change? I'm not sure if this is change we wanted to make.

Reply:
no dedicated issue for this, I discussed it with @kepano here: https://github.com/obsidianmd/obsidian-importer/issues/204#issuecomment-1915558518

Reply:
Ah great, thank you.